alloc-customization-point.html

```diff
@@ -90,16 +93,48 @@ <h2>tl;dr</h2>
     std::pmr::polymorphic_allocator&lt;&gt;&gt;::type>
 : BloombergLP::bslma::UsesBslmaAllocator&lt;TYPE&gt;::type
 {
-  static BloombergLP::bslma::Allocator* rebind(std::pmr::polymorphic_allocator&lt;&gt; a)
+  static BloombergLP::bslma::Allocator* rebind(std::pmr::polymorphic_allocator&lt;&gt;const& a)
```

Comment:
Please don't change allocator parameters from by-value to by-reference, allocators tend to be passed by value.